### PR TITLE
Added my profile markup

### PR DESCRIPTION
**Pull request by cValentyna**
Added my profile markup. I tried to send my first level by smaller parts. This is first part.  
I just noticed that something wrong with my fontWeight.

### CODE REVIEW

**Review by 1937Carrie — CHANGES_REQUESTED** [review]

File: app/src/main/res/drawable/full_fb.xml
Comment (cValentyna) [inline app/src/main/res/drawable/full_fb.xml:1]: connected ellips with logo

File: app/src/main/res/font/open_sans_semibold.ttf
Comment (cValentyna) [inline app/src/main/res/font/open_sans_semibold.ttf:1]: added one more type of font

File: app/src/main/res/layout-land/activity_main.xml
Comment (cValentyna) [inline app/src/main/res/layout-land/activity_main.xml:1]: added landscape orientation (changed only image size)

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -2,18 +2,197 @@
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (cValentyna) [inline app/src/main/res/layout/activity_main.xml:2]: returned constraint layout

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -2,18 +2,197 @@
...
-    android:id="@+id/main"
+    android:id="@+id/activity_main"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="@color/secondary_background"
+    android:theme="@style/Theme.MyApp"
     tools:context=".MainActivity">
 
-    <TextView
+    <View
```
Comment (cValentyna) [inline app/src/main/res/layout/activity_main.xml:12]: added view for blue part

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -2,18 +2,197 @@
...
+        android:id="@+id/v_main_top"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:background="@color/primary_background"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintHeight_percent="0.5"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0" />
+
+    <androidx.appcompat.widget.AppCompatTextView
```
Comment (cValentyna) [inline app/src/main/res/layout/activity_main.xml:22]: changed types of views from simple to appcompat

File: app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -2,18 +2,197 @@
...
+        android:text="@string/main_log_out"
+
+        app:layout_constraintBottom_toBottomOf="@id/tv_main_settings"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="@id/tv_main_settings"
+        app:layout_constraintTop_toTopOf="@id/tv_main_settings"
+        app:layout_constraintWidth_max="@dimen/width_log_out_button_max"
+        app:layout_constraintWidth_percent="@dimen/width_percent_log_out_button" />
+
+    <com.google.android.material.imageview.ShapeableImageView
```
Comment (cValentyna) [inline app/src/main/res/layout/activity_main.xml:56]: removed default size of image. cardview was changed to ShapeableImageView

File: app/src/main/res/layout/activity_main.xml [hunk 5]
```diff
@@ -2,18 +2,197 @@
...
+        android:layout_marginEnd="@dimen/margin_social_networks"
+
+        android:src="@drawable/full_fb"
+
+        app:layout_constraintBottom_toTopOf="@id/bt_main_edit"
+        app:layout_constraintEnd_toStartOf="@id/iv_main_linkedin"
+        app:layout_constraintHorizontal_chainStyle="packed"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/v_main_top"
+        app:layout_constraintVertical_bias="@dimen/bias_vertical_social_network_frame" />
```
Comment (cValentyna) [inline app/src/main/res/layout/activity_main.xml:133]: imageButtos were changet to ImageView.

File: app/src/main/res/layout/activity_main.xml [hunk 6]
```diff
@@ -2,18 +2,197 @@
...
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/tv_main_user_job" />
+
+    <androidx.appcompat.widget.AppCompatImageView
+        android:id="@+id/iv_main_facebook"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="@dimen/margin_social_networks"
+
+        android:src="@drawable/full_fb"
```
Comment (cValentyna) [inline app/src/main/res/layout/activity_main.xml:126]: Elipses and logos are together

File: app/src/main/res/values-v26/styles.xml
Comment (cValentyna) [inline app/src/main/res/values-v26/styles.xml:1]: added fontWeight for v26

File: app/src/main/res/values-v28/styles.xml
Comment (cValentyna) [inline app/src/main/res/values-v28/styles.xml:1]: Added styles for different weights when 28 and higher api is used. Unfortunately it does not work when dimension is used.

File: app/src/main/res/values/dimens.xml
Comment (cValentyna) [inline app/src/main/res/values/dimens.xml:1]: renamed names and added needed values

File: app/src/main/res/values/styles.xml [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+
+    <style name="TextAppearance_bt_edit_profile" parent="primary_text_style">
+        <item name="android:textColor">@color/primary_background</item>
+        <item name="android:textSize">@dimen/textsize_medium</item>
+    </style>
+
+    <style name="TextAppearance_bt_contact" parent="text_big_button_style">
+        <item name="android:textColor">@color/the_brightest_font</item>
+    </style>
+<!-- style for image frame   -->
```
Comment (cValentyna) [inline app/src/main/res/values/styles.xml:46]: style for image shape

**Review by 1937Carrie — APPROVED** [review]
Correct margin of text profession to the top and to the bottom and go on.

**Review by EugeneOrlov204 — APPROVED** [review]

**Pull request merged**